skore-ui/src/components/ProjectViewCard.vue

```diff
@@ -20,16 +38,35 @@ const emit = defineEmits<{
         <div class="title">{{ props.title }}</div>
         <div class="subtitle" v-if="props.subtitle">
           {{ props.subtitle }}
+          <Transition name="fade">
+            <span v-if="!isLatestUpdate" class="warning">
+              <i class="icon-warning"></i>
```

Comment:
The little warning symbol after "Created x minutes ago" is so small as to be indistinguishable IMO. I would put the "You are viewing an old update" on a separate line.

Reply:
Adding a second line is in my opinion not a good idea as it will lead to inconsistency in card height hence generating a scroll of everything below. Would a warning symbol without the circle do the job ?

Reply:
Gotcha. Yeah, I think it would work better!

Reply:
Icon has been changed.